array_api_tests/hypothesis_helpers.py

```diff
@@ -121,8 +125,17 @@ def two_broadcastable_shapes(draw, shapes=shapes):
 
 nonbroadcastable_ones_array_two_args = hypotheses_tuples(ones_arrays, ones_arrays)
 
-# TODO: Generate general arrays here, rather than just scalars.
-numeric_arrays = builds(full, just((1,)), floats())
+floating_arrays = xps.arrays(dtype=xps.floating_dtypes(), shape=xps.array_shapes())
+
+@composite
+def broadcastable_floating_array_pairs(draw):
+    dtype = draw(xps.floating_dtypes())
+    broadcastable_shapes = draw(xps.mutually_broadcastable_shapes(2, min_dims=1))
```

Comment:
Why min_dims=1? We definitely want to make sure dimension 0 arrays are tested. 

Actually, can't you use two_mutually_broadcastable_shapes defined above here?

Reply:
No idea why I did this heh.